research/neural_programmer/neural_programmer.py

```diff
@@ -113,9 +113,9 @@ def evaluate(sess, data, batch_size, graph, i):
                                                             graph))
     gc += ct * batch_size
     num_examples += batch_size
-  print "dev set accuracy   after ", i, " : ", gc / num_examples
-  print num_examples, len(data)
-  print "--------"
+  print("dev set accuracy   after ", i, " : ", gc / num_examples)
+  print(num_examples, len(data))
+  print("--------")
```

Comment:
Same here--we need `from __future__ import print_function`.

Reply:
Thanks!  Done.

Reply:
Excellent!